### PR TITLE
T129 align flash

### LINKED ISSUE #129: [NFR]: Flash - align with cphalcon
Align `Phalcon\Flash\*` with changes from `cphalcon`

### PR DESCRIPTION
**Pull request by niden**
Aligning Flash with cphalcon

### CODE REVIEW

File: src/Html/Link/Serializer/Header.zep [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
```
Comment (Jeckerson) [inline src/Html/Link/Serializer/Header.zep:1]: Filename has `.zep` extension

File: src/Html/Link/Serializer/SerializerInterface.zep [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (Jeckerson) [inline src/Html/Link/Serializer/SerializerInterface.zep:1]: Filename has `.zep` extension

Comment (niden) [reply src/Html/Link/Serializer/SerializerInterface.zep:1]: Obsolete files.

File: src/Html/Link/Serializer/Header.zep (hunk 1 above)
Comment (niden) [reply src/Html/Link/Serializer/Header.zep:1]: Obsolete files.

**Review by Jeckerson — APPROVED** [review]

**Pull request merged**